src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -60,6 +61,9 @@ export interface ShippingState {
   errorModalTitle: string
   errorModalMessage: string
 }
+
+const logger = createLogger("Shipping/index.tsx")
```

Comment:
Same here, we're a little inconsistent but might as well standardize while we're in here.